tests/test_geometry.scad

```diff
@@ -987,8 +987,8 @@ module test_pointlist_bounds() {
 
 
 module test_closest_point() {
-    ptlist = [for (i=range(100)) rands(-100,100,2,seed_value=8463)];
-    testpts = [for (i=range(100)) rands(-100,100,2,seed_value=6834)];
+    ptlist = [for (i=count(100)) rands(-100,100,2,seed_value=8463)];
+    testpts = [for (i=count(100)) rands(-100,100,2,seed_value=6834)];
```

Comment:
This code produces 100 copies of the same point.  If that's the intent, change to use repeat().  If not the intent....fix.

Reply:
Would adding `i` to the seed value be good enough?

Reply:
I think so.  Could also use array_group.  testpts = array_group(rands(-100,100,200,seed_value=33),2);